src/actions/ScanActions.js

```diff
@@ -154,8 +154,9 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
     if (isLegacyAddressUri(parsedUri)) {
       // LEGACY ADDRESS URI
       if (await shouldContinueLegacy()) {
+        const guiMakeSpendInfo = { ...parsedUri }
```

Comment:
Change to `const guiMakeSpendInfo: GuiMakeSpendInfo = { ...parsedUri }`, to be explicit about the type conversion. You will also need to `import { type GuiMakeSpendInfo } from '../types/types.js'`